views/src/Pages/Checklist.jsx

```diff
@@ -0,0 +1,291 @@
+                  </Table.Th>
+                  <Table.Th style={{ textAlign: "center", width: "10%" }}>
+                    React
+                  </Table.Th>
+                  <Table.Th style={{ textAlign: "center", width: "10%" }}>
+                    SQL
+                  </Table.Th>
+                  <Table.Th style={{ textAlign: "center", width: "10%" }}>
+                    ExpressJS
+                  </Table.Th>
+                  <Table.Th style={{ textAlign: "center", width: "10%" }}>
+                    API Integration
+                  </Table.Th>
+                </Table.Tr>
+              </Table.Thead>
+            </Table>
+            <Table
+              highlightOnHover={true}
+              striped={true}
+              stickyHeader={true}
+              w="100%"
+              withTableBorder={true}
+              withColumnBorders={true}
+              withRowBorders={true}
+            >
+              <Table.Thead>
+                <Table.Tr>
+                  <Table.Th id="htmlSubmitted" style={{ width: "8.3333333%" }}>
+                    Submitted
+                  </Table.Th>
+                  <Table.Th id="htmlCompleted" style={{ width: "8.3333333%" }}>
+                    Completed
+                  </Table.Th>
+                  <Table.Th id="htmlReviewer" style={{ width: "calc(25%/3)" }}>
+                    Reviewer
+                  </Table.Th>
+                  <Table.Th id="cssSubmitted" style={{ width: "calc(25%/3)" }}>
+                    Submitted
+                  </Table.Th>
+                  <Table.Th id="cssCompleted" style={{ width: "calc(25%/3)" }}>
+                    Completed
+                  </Table.Th>
+                  <Table.Th id="cssReviewer" style={{ width: "calc(25%/3)" }}>
+                    Reviewer
+                  </Table.Th>
+                  <Table.Th id="mdrqSubmitted" style={{ width: "calc(25%/3)" }}>
+                    Submitted
+                  </Table.Th>
+                  <Table.Th id="mdrqCompleted" style={{ width: "calc(25%/3)" }}>
+                    Completed
+                  </Table.Th>
+                  <Table.Th id="mdrqReviewer" style={{ width: "calc(25%/3)" }}>
+                    Reviewer
+                  </Table.Th>
+                  <Table.Th
+                    id="bootstrapSubmitted"
+                    style={{ width: "calc(25%/3)" }}
+                  >
+                    Submitted
+                  </Table.Th>
+                  <Table.Th
+                    id="bootstrapCompleted"
+                    style={{ width: "calc(25%/3)" }}
+                  >
+                    Completed
+                  </Table.Th>
+                  <Table.Th
+                    id="bootstrapReviewer"
+                    style={{ width: "calc(25%/3)" }}
+                  >
+                    Reviewer
+                  </Table.Th>
+                  <Table.Th id="jsSubmitted" style={{ width: "calc(25%/3)" }}>
```

Comment:
Same thing as in the other comment. Instead of adding the styles directly to the element, why not create a style variable and feed that to the style property. That way it's much cleaner. so something like below

```
                  const styleForElement = {
         display: "flex",
        flexDirection: "column-reverse",
        position: "relative",
    } 
    
    <Table.Th  style={styleForElement }>My HEADING</Table.Th>
```

Reply:
Will check.